docs/config.yml

```diff
@@ -31,3 +31,9 @@ security:
     - '^HUGO_'
     - 'VERSIONS'
     - 'CURRENT_VERSION'
+
+go_initialisms:
+  replace_defaults: true
```

Comment:
People often copy and paste this document, so we should default to false to encourage the default.

Reply:
Done!